config/etcd-io/sig-etcd/teams.yaml

```diff
@@ -111,6 +111,7 @@ teams:
     - siyuanfoundation
     - thedtripp
     - tjungblu
+    - henrybear327
```

Comment:
I think the linter will fail because the list is not alphabetically ordered :see_no_evil:

Reply:
Oh no...
Fixing now! Thanks!